bundles/org.openhab.binding.modbus.sunspec/src/main/feature/feature.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <features name="org.openhab.binding.modbus.sunspec-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
-    <repository>file:${basedirRoot}/bundles/org.openhab.binding.modbus/target/feature/feature.xml</repository>
+    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
 
-    <feature name="openhab-binding-modbus-sunspec" description="SunSpec Bundle" version="${project.version}">
+    <feature name="openhab-binding-modbus-sunspec" description="Modbus Binding SunSpec" version="${project.version}">
         <feature>openhab-runtime-base</feature>
-        <feature>openhab-binding-modbus</feature>
+        <feature>openhab-transport-modbus</feature>
+        <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.modbus/${project.version}</bundle>
```

Comment:
This is different from what we do in Bluetooth and mqtt. There we have no feature.xml at all for the bundles but add the aggregated feature in the feature-build-bundle (IIRC footer.xml).

Reply:
What is this then? https://github.com/openhab/openhab-addons/blob/2.5.x/bundles/org.openhab.binding.bluetooth.blukii/src/main/feature/feature.xml

Reply:
And https://github.com/openhab/openhab-addons/blob/2.5.x/bundles/org.openhab.binding.bluetooth.am43/src/main/feature/feature.xml and https://github.com/openhab/openhab-addons/blob/2.5.x/bundles/org.openhab.binding.bluetooth.airthings/src/main/feature/feature.xml ...

Reply:
features/openhab-addons/src/main/resources/footer.xml 

This are the features that are installed, so we should add modbus here, too. The single-features are excluded here: https://github.com/openhab/openhab-addons/blob/202bca75ed9db8ec64677a384ca1da3bd92e44a9/features/openhab-addons/pom.xml#L47

Reply:
Ok, I will add modbus there as well.
But what I don't understand yet: How/where are the aggregated features generated?

Reply:
It slowly all comes back :-). If you tell me where to put it, I‘ll add a section on feature-definition and aggregation to the developer documentation. Since this is a bit special, I‘m not sure if it should be part of the general description for binding development.

Reply:
> Ok, I will add modbus there as well.

Done.

>  If you tell me where to put it

Hm, tricky question. We don't have anything about the general structure for an add-on yet.
https://www.openhab.org/docs/developer/buildsystem.html comes closest - maybe some info on feature definition could be added to that page, wdyt?

Reply:
Added. I already wrote an explanation for the new embedding-strategy for dependencies but forgot to create a PR for that. Shame on me.

Reply:
Thanks!